### DISCUSSION TITLE
Unable to find server files on the shared folder of Dsm7.2

### DISCUSSION THREAD

**Discussion by xpmod** [itzg/docker-minecraft-server#2701] [Q&A]
I mounted the container to a shared folder on Dsm7.2, but I couldn't find the server's corresponding files in the mounted shared folder. What should I do when creating a container to display the server files on the shared folder.
I didn't have this situation when using Docker in Dsm 7.1 before, but after updating to 7.2, Docker changed to Container Manager, which caused this situation.

**Comment by itzg** [comment]
Please provide more specifics.

**Reply by xpmod** [reply]
I chose to mount it to the doc/data folder, and after waiting for the server to open, I did not see any files about the server in the folder I mounted
![屏幕截图 2024-03-10 222556](https://github.com/itzg/docker-minecraft-server/assets/126975613/48587d31-26b4-49e2-a2bf-890c61f73c87)
![屏幕截图 2024-03-10 222614](https://github.com/itzg/docker-minecraft-server/assets/126975613/aeebcc74-778b-469c-b90b-7f91504819b8)

**Reply by itzg** [reply]
You mounted the host path `/doc`

![image](https://github.com/itzg/docker-minecraft-server/assets/988985/2c72f389-3843-4d98-b17e-5ef476b94e40)

So you need to be looking in `/doc` which is an absolute path. In the file browser screenshot I can't tell what path that is exactly and there won't be a `data` subdirectory.
